docs/SNAPTELD.md

```diff
@@ -52,6 +52,7 @@ $ snapteld [global options] command [command options] [arguments...]
 --rest-cert value                            A path to a certificate to use for HTTPS deployment of Snap's REST API
 --rest-key value                             A path to a key file to use for HTTPS deployment of Snap's REST API
 --rest-auth                                  Enables Snap's REST API authentication
+--rest-load-path                             A configurable path from which the API loads plugins
```

Comment:
We should consider exposing this as top level (control) config.  It could be called `--temp-dir-path`.  This will line up when we resolve #1502.